metabase/metabase
# Update collection empty state

Epic https://github.com/metabase/metabase/issues/23222

Changes:
- Extracts the menu for creating new items from the top nav
- Uses this menu for creating new items in the nav, collection header, and collection empty state
- Updates the empty state UI

How to test:
- Open an empty collection
- Make sure that both the top right new button, new button in the collection header and the empty state work correctly

<img width="913" alt="Screenshot 2022-06-17 at 14 48 34" src="https://user-images.githubusercontent.com/8542534/174292677-1dd56bc0-7545-41bf-b9de-14ab99dce5b7.png">
<img width="909" alt="Screenshot 2022-06-17 at 14 48 47" src="https://user-images.githubusercontent.com/8542534/174292692-a333156f-8bc6-4dd2-974c-dff05025f9fd.png">
